xarray/core/rolling.py

```diff
@@ -282,6 +278,12 @@ def wrapped_func(self, **kwargs):
             padded = self.obj.variable
             if self.center:
                 shift = (-self.window // 2) + 1
+
+                if (LooseVersion(np.__version__) < LooseVersion('1.13') and
```

Comment:
I notice that with the older numpy and benchmark, reducing of `np.nan`-and-boolean-mixed arrays gives a wrong result.
Here is a workaround, where we cast boolean arrays to float.